code/modules/client/preference/preferences_mysql.dm

```diff
@@ -495,3 +501,20 @@
 	load_character(C,pick(saves))
 	return 1
 
+/datum/preferences/proc/clear_character_slot(client/C)
+	. = FALSE
+	// Is there a character in that slot?
+	var/DBQuery/query = GLOB.dbcon.NewQuery("SELECT slot FROM [format_table_name("characters")] WHERE ckey='[C.ckey]' AND slot='[default_slot]'")
```

Comment:
Please sanitize the ckey. I know its unlikely, but never underestimate the chance that someone joins called `'; DROP DATABASE 'paradise`

Reply:
I agree with the idea that you should **always** sanitize anything you pass to SQL, but as we found out, it isn't needed here (or in any other SQL procs present in the file)

`world.log << ckey("'; DROP DATABASE 'db")` outputs `dropdatabasedb`